gflownet/proxy/crystals/lattice_parameters.py

```diff
@@ -0,0 +1,32 @@
+import pickle
+from pathlib import Path
+from typing import List
+
+import numpy as np
+from torch import Tensor
+
+from gflownet.proxy.base import Proxy
+
+PICKLE_PATH = Path(__file__).parents[3] / "data" / "crystals" / "lattice_proxy.pickle"
+
+
+class LatticeParameters(Proxy):
+    def __init__(self, min_value: float = -100, **kwargs):
+        super().__init__(**kwargs)
+
+        self.min_value = min_value
+        self.kde = None
+
+    def __call__(self, states: List) -> Tensor:
+        scores = self.kde.score_samples(states)
+        scores = np.clip(scores, self.min_value, np.inf)
+        scores = self.min_value - scores
+
+        return Tensor(scores)
```

Comment:
To follow the standard in the repo:
```suggestion
        return tfloat(scores, float_type=self.float, device=self.device)
```